README.md

```diff
@@ -15,24 +15,15 @@ A Cornell University CIT Custom Development starter kit package for Laravel.
    ```shell
    composer create-project laravel/laravel example-app
    ```
-2. Add the CD-LaravelStarterKit repository to `composer.json`:
-   ```
-   "repositories": [
-     {
-       "type": "vcs",
-       "url": "git@github.com:CU-CommunityApps/CD-LaravelStarterKit.git"
-     }
-   ],
-   ```
-3. Composer require the CD-LaravelStarterKit
+2. Composer require the LaravelStarterKit
    ```shell
-   composer require cu-communityapps/cd-laravelstarterkit
+   composer require cornell-custom-dev/laravel-starter-kit
```

Comment:
I am no sure I understand the instruction. I pulled the code from the  `html-css-basics` branch. The instaruction says to run `composer require cornell-custom-dev/laravel-starter-kit:dev-html-css-basics`
that gives an error: `Root package 'cornell-custom-dev/laravel-starter-kit' cannot require itself in its composer.json`

Reply:
Oh! It looks like this is for usage - for a final laravel project. Does it mean we have to add css framework as a separate package and we are not going to include it into the starter kit?

Reply:
Yes, this is for how to pull in the starter kit into a new Laravel project. The css framework is brought in as part of `composer require cornell-custom-dev/laravel-starter-kit`, and the assets are installed with `artisan starterkit:install`. I think your questions point out that I need to clarify things more in the differences between the usage instructions of the README and the development / testing instructions of CONTRIBUTING. I will work on that.